sushibar-backend-app/src/main/java/org/virtualsushibar/backend/app/dao/document/OrderStatus.java

```diff
@@ -0,0 +1,7 @@
+package org.virtualsushibar.backend.app.dao.document;
+
+public enum OrderStatus {
+    ORDER_CONFIRMED,
```

Comment:
We can add a status as Pending order, for the retry attempt

Reply:
there is already this: ORDER_NOT_CONFIRMED